### ISSUE TITLE
Remove Load more Textview from the Expandable List activities

### ISSUE THREAD

**Issue by avidraghav** [malliaridis/Qsol#4]
### What it does?
It moves to an Activity which shows an animation for few seconds then returns back to the Expandable List Activity

### What was the purpose of implementing it?
When the Expandable list activities are opened for the first time the counter values are not displayed correctly (the reason as suggested: the location of database is very far from where the project is used, hence the delay in loading the correct values). The Load more button just reopens the same activity after a delay of few seconds so as the correct counter values can be loaded.

### What has to be done?
 Remove the Load more Textview from all Expandable list Activities then remove the files which become useless after this change such as xml files, Github library to load an animation etc.

### How should be the final output?
As the new database's location is relevant to the location from where the project is used, the counters should display the exact values as soon as the activity opens.

**Comment by malliaridis** [comment]
Assignees should work on this after #2 is merged into `dev`.
